### ISSUE TITLE
Need Dashboard

### ISSUE THREAD

**Issue by ahsan3219** [FlowiseAI/Flowise#2434]
I need a dashboard where I can analyze bot conversations with users. It also provides the ability to connect the bot at any time with a live human agent.

**Comment by nich2198** [comment]
Have you found any? Looking for the same

**Comment by BigFoxMedia** [comment]
can you guys provide a more detailed description of what you need? features, ui, etc.
I'm working on a project and might help.

**Comment by nich2198** [comment]
Yeah, well i run my chatbot via Flowise API. I need a dashboard that can follow the conversations made on the chatbot for my clients to see. My chatbot is made with react.js and the API and so on. But something like https://chat-dash.com/, bit for flowise.
